### ISSUE TITLE
Implemented on other models

### ISSUE THREAD

**Issue by JingyiChen1996** [google-research/mixmatch#26]
Hi, 
Thank you for the amazing paper and sharing the code! I wondered did you try this MixMatch method on other models (e.g. DNN121) and other datasets? I implemented this method on CIFAR10 using DNN121, but the result is much worse than using WideResNet. I also implemented this method on medical images, but the result is worse than PI model and mean teacher. 
I'm sure this is probably due to different hyperparameters. I tried different sets of learning rates and lamba_u, but it didn't improve the result much. If possible, could you provide some tips on how to tune the parameters for different models? 

Thank you for your help in advance!

**Comment by david-berthelot** [comment]
1. I am not familiar with DNN121, I noticed that batch norm was important for the method to work well. So if this model doesn't use batch norm, that might explain your issue.
2. On medical images, we haven't tried.
3. Did you use our code or reimplement it from the paper description (also note in our code there's no lamba_u, there's w_match)?
As an experiment I would recommend just loading your data in our codebase and see what you get.
